package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-zapier",
-  "version": "5.0.0",
+  "version": "6.0.0",
```

Comment:
Since this change would not introduce new errors, but only potentially reduce the number of errors, we can do a minor release for this instead.
We only need a major release if modifying an existing rule would introduce more errors (because it would break people's tests). I'll update the README to make this more clear!

Reply:
PR here: https://github.com/zapier/eslint-plugin-zapier/pull/29